sdk/translation/azure-ai-translation-text/src/test/java/com/azure/ai/translation/text/BreakSentenceTests.java

```diff
@@ -17,7 +18,8 @@ public class BreakSentenceTests extends TextTranslationClientBase {
     public void breakSentenceWithAutoDetect() {
         BreakSentenceItem response = getTranslationClient().findSentenceBoundaries("hello world");
         assertEquals("en", response.getDetectedLanguage().getLanguage());
-        assertEquals(0.98, response.getDetectedLanguage().getConfidence());
+        assertTrue(response.getDetectedLanguage().getConfidence() > 0.8); //Created task: https://machinetranslation.visualstudio.com/MachineTranslation/_workitems/edit/164493
```

Comment:
```suggestion
        assertTrue(response.getDetectedLanguage().getConfidence() > 0.8);
```

Reply:
Updated, here are the live test pipeline results: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4098377&view=ms.vss-test-web.build-test-results-tab